draft-ietf-tls-esni.md

```diff
@@ -49,7 +49,7 @@ encrypting a ClientHello message under a server public key.
 
 --- middle
 
-# Introduction
+# Introduction {#intro}
```

Comment:
I think the term "anonymity set" needs to be more clearly defined here. It warrants a short paragraph.

Reply:
It's defined where it's used.

Reply:
Yeah, better to put it where it's first defined. I guess my issue is how it's defined. It's not that they have the same configuration ... what matters is simply that the ClientHello protection service is provided on their behalf. Maybe replace "wherein all elements have a consistent configuration..." with "of servers for which the service is provided".

Reply:
Chatted offline -- will improve the definition in a separate issue!